test/.jshintrc

```diff
@@ -13,7 +13,7 @@
     "maxparams": 6,
     "maxdepth": 4,
     "camelcase": true,
-    "maxlen": 120,
+    "maxlen": 180,
```

Comment:
Why this change is needed?

I mean, if prettier is ensuring `"printWidth": 120`, why the linter needs to extend +60 chars the limit?

Just curiosity...

Reply:
@fgalan, prettier is *attempting* to fit within 120 characters, but according to its own opinionated
rules it won't always succeed particularly if you have a line with a long string. Prettier will **always** remove unnecessary concatenations in strings, and always indent properly (with a tab 4) so if you are left with a line over 120 characters long jslint will complain.

We're fortunate that we never need very long strings in the code base then. However each of the tests **does** contain a  single lengthy  string in it - the description of the test itself - also this
ends up being prettily indented by about 4 tabs. 

Rather than "fixing" this by truncating the description text, I chose to relax the jslint rule for the **tests only**.

Of course if you leave the default tab at 2 spaces you're less likely to reach the limit.

Reply:
I think is a good solution. No problem at all. Thanks!